configs/sassd/sassd_6x8_80e_kitti-3d-3class.py

```diff
@@ -0,0 +1,5 @@
+_base_ = [
+    '../_base_/models/sassd.py',
```

Comment:
We suggest directly adding model-related configs in this file instead of adding another new base model config.